src/components/ArticlesList/ArticlesList.jsx

```diff
@@ -0,0 +1,54 @@
+// imports
+import { useEffect, useState } from "react";
+import exampleArticlesData from "../../SampleData/api-articles.json";
+import ArticleCard from "./ArticleCard";
+import axios from "axios";
+
+// Creating a custom axios instance
+const apiClient = axios.create({
+  baseURL: "https://my-backend-project-bc-news.onrender.com",
+  timeout: 1000,
+});
+//
+function ArticlesList() {
+  const [articles, setArticles] = useState([]);
+  const [loading, setLoading] = useState(true); // was getting an error, return in the jsx below would not render, this is because the render was happening before the data was fetched from the api, instead I put in a conditional, when it loads it will then show!
+
+  function helperFunction() {
+    console.log(articles, "useState");
+  }
+  helperFunction();
```

Comment:
This is interesting, but should be pushed up to github if it was used as a debugging tool